tests/yam/agama/boot_agama.pm

```diff
@@ -22,6 +23,15 @@ BEGIN {
 use bootloader_s390;
 use bootloader_zkvm;
 
+sub expand_profile {
+    my ($self, $profile) = @_;
+    my $content = expand_variables(get_test_data($profile));
+    my $profile_expanded = 'profile.json';
```

Comment:
the filepath should not be generic, but the one introduced by setting.

Reply:
Done